.github/workflows/consuming.yml

```diff
@@ -18,9 +18,14 @@ jobs:
       matrix:
         project: ['admiral', 'submariner', 'submariner-operator', 'lighthouse']
         deploytool: ['operator', 'helm']
+        cable_driver: ['libreswan', 'strongswan', 'wireguard']
```

Comment:
I hadn't noticed that we make the test matrix quite explode.

I propose we test with only one cable driver, other than the default, `wireguard`?, that way we can always check that such driver is being used.

Reply:
I see no reason, in this particular case, to specify it (as it'll double the amount of jobs).

Instead, I propose a simple "unit test" style test which will see that this was indeed passed properly.
Alternatively, for the time being we can just not test it here, since it is being tested by other projects

Reply:
I mean, just having one which won't be the default may be?

But I agree that we cannot grow the test matrix, the e2e is by no ways a cheap test to run.

Reply:
By this reasoning, why do we test different deploy tools? They are already covered in other repos and they expand the test matrix.